### ISSUE TITLE
Multiple axes

### ISSUE THREAD

**Issue by mohamadelhinamy** [indiespirit/react-native-chart-kit#482]
Can I have a graph with one x-axis and multiple y-axis.
What I want to achieve is have a graph that combines multiple graphs together and measure different values say volts and amperes for instance is that possible to have multiple datasets and each one has its own y-axis but all in the same graph?

**Comment by dhameliyawork** [comment]
Hi, is there any support for multple Y axis on same graph. 

I am looking to achieve something like this


![Screenshot 2024-01-10 at 6 31 22 PM](https://github.com/indiespirit/react-native-chart-kit/assets/104249713/8008f7ca-613f-498d-ab37-695a362d6a8e)

**Comment by prajjwal-23** [comment]
Yes, you can use react-native-gifted-charts to display multiple datasets on a single chart, including support for dual Y-axes. This allows you to visualize different metrics.

The library offers features like:

- Multiple data series using data and data2 props
- Dual Y-axis rendering (left and right)
- Custom styling for each dataset (colors, area fill, suffixes)
- Smooth, curved lines and area charts

It’s a flexible and well-supported solution for advanced charting in React Native (including Expo environments).
